pyproject.toml

```diff
@@ -1,13 +1,14 @@
 [build-system]
-requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
+build-backend = "poetry_dynamic_versioning.backend"
 
 [tool.poetry]
 name = "imap-processing"
-version = "0.2.0"
+version = "0.0.0"
 description = "IMAP Science Operations Center Processing"
 authors = ["IMAP SDC Developers <imap-sdc@lists.lasp.colorado.edu>"]
 readme = "README.md"
+include = ["imap_processing/_version.py"]
```

Comment:
Note that we will be producing a `_version.py` file now, but it is gitignored so that will be dynamically updating everytime we install but not committed to the repo.

Reply:
Just curious. How often do we need to install to update this? Is this done per release?

Reply:
Every time you install the project it will update this from the latest tag/commit for you. You never have to touch this, it is all automated. If we tag on release, it should pick up that tag and set that as the version number.